differ/collectionsDiffer.go

```diff
@@ -11,25 +11,24 @@ func NewDefaultCollectionsDiffer() *defaultCollectionsDiffer {
 	return &defaultCollectionsDiffer{}
 }
 
-func (dcd *defaultCollectionsDiffer) Diff(incomingCollectionUuids []string, oldCollectionUuids []string) (map[string]bool) {
-	diffCol := make(map[string]bool)
+func (dcd *defaultCollectionsDiffer) Diff(incomingCollectionUuids []string, oldCollectionUuids []string) *Set {
```

Comment:
This func should probably be called `SymmetricDifference` with the `oneWayDiff` being renamed to `setDifference` to match standard set theory naming conventions